### PR TITLE
Fixes #28121 -  Bump fog-ovirt to 1.2.3

### PR DESCRIPTION
**Pull request by shiramax**

### CODE REVIEW

File: q [hunk 1]
```diff
@@ -0,0 +1,63 @@
+=>   <[1;34;4mFog[0m::[1;34;4mOvirt[0m::[1;34;4mCompute[0m::[1;34;4mServer[0m
```
Comment (tbrisker) [inline q]: I don't think this file belongs in the commit :)

File: bundler.d/ovirt.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 group :ovirt do
-  gem 'fog-ovirt', '~> 1.2.0'
+  # gem 'fog-ovirt', '~> 1.2.0'
+  gem 'fog-ovirt', :path => '../fog-ovirt'
```
Comment (tbrisker) [inline bundler.d/ovirt.rb]: probably not what you planned to commit?

**Review by tbrisker — DISMISSED** [review]

File: bundler.d/ovirt.rb (hunk 2 above)
Comment (shiramax) [reply bundler.d/ovirt.rb]: @tbrisker sorry! fixed it :)

**Review by evgeni — DISMISSED** [review]

**Comment by evgeni** [comment]
packaging is merged, so this can be merged now

**Comment by ezr-ondrej** [comment]
Hope @tbrisker won't get angry, but I believe his comments were addressed.

**Review by ezr-ondrej — APPROVED** [review]
Thanks @shiramax!

**Comment by ezr-ondrej** [comment]
@evgeni could we please get another ack?

**Review by evgeni — APPROVED** [review]
still ack,

test failures are blocked on https://projects.theforeman.org/issues/28406 and unrelated

**Comment by ezr-ondrej** [comment]
As @evgeni said, failures are unrelated, merging.

**Pull request merged**